### DISCUSSION TITLE
BardAsync lacks Session? (ops its already working)

### DISCUSSION THREAD

**Discussion by orsnaro** [dsdanielpark/Bard-API#86] [Q&A]
~Great work my fellow ! is there any chance to add Bard chat  sessions to `BardAsync` any time soon?~

edit:
seems to be already working . my fault didn't test it enough I've read core sync  and compared it to core.py now 

t.c 💙

**Comment by dsdanielpark** [comment]
BardAsync has not undergone thorough QA testing yet. If you encounter any issues, please report them or submit a pull request with fixes. Thank you for your cooperation.
